modules/sparql/src/test/scala/ch/epfl/bluebrain/nexus/kg/indexing/instances/InstanceSparqlIndexerSpec.scala

```diff
@@ -66,18 +66,19 @@ class InstanceSparqlIndexerSpec(blazegraphPort: Int)
 
   private val base                   = s"http://$localhost/v0"
   private val blazegraphBaseUri: Uri = s"http://$localhost:$blazegraphPort/blazegraph"
+  private val namespace              = genString(length = 6)
 
-  private val settings @ InstanceSparqlIndexingSettings(index, instanceBase, _, nexusVocBase) =
-    InstanceSparqlIndexingSettings(genString(length = 6), base, s"$base/data/graphs", s"$base/voc/nexus/core")
+  private val settings @ InstanceSparqlIndexingSettings(instanceBase, baseNs, nexusVocBase) =
+    InstanceSparqlIndexingSettings(base, s"$base/data/graphs", s"$base/voc/nexus/core")
 
   private implicit val stringQualifier: ConfiguredQualifier[String]         = Qualifier.configured[String](nexusVocBase)
   private implicit val orgIdQualifier: ConfiguredQualifier[OrgId]           = Qualifier.configured[OrgId](base)
   private implicit val domainIdQualifier: ConfiguredQualifier[DomainId]     = Qualifier.configured[DomainId](base)
   private implicit val schemaIdQualifier: ConfiguredQualifier[SchemaId]     = Qualifier.configured[SchemaId](base)
   private implicit val schemaNameQualifier: ConfiguredQualifier[SchemaName] = Qualifier.configured[SchemaName](base)
 
-  private def triples(client: SparqlClient[Future]): Future[List[(String, String, String)]] =
-    client.query(index, s"SELECT * WHERE { ?s ?p ?o }").map { rs =>
+  private def triples(id: InstanceId, client: SparqlClient[Future]): Future[List[(String, String, String)]] =
```

Comment:
Shouldn't we check on every graph, in order to make sure blazegraph is not creation triples out of the graph?

Reply:
done